configure.ac

```diff
@@ -4297,7 +4305,11 @@ dnl BeOS support removed after SDL 2.0.1. Haiku still works.  --ryan.
         SOURCES="$SOURCES $srcdir/src/video/uikit/*.m"
         SUMMARY_video="${SUMMARY_video} uikit"
         have_video=yes
-        EXTRA_LDFLAGS="$EXTRA_LDFLAGS -lm -liconv -lobjc"
+        EXTRA_LDFLAGS="$EXTRA_LDFLAGS -lm"
+        if test x$enable_libc = xyes; then
```

Comment:
You probably meant $enable_iconv here?

Reply:
Yes yes yes. So sorry!
Fixed in https://github.com/libsdl-org/SDL/commit/5d62b3bdb0a216732b2aae63015e3b2a014ec865